### ISSUE TITLE
 'BatchNorm2d' object has no attribute 'track_running_stats'. 

### ISSUE THREAD

**Issue by olivia93c** [norawebbwilliams/images_as_data#2]
I keep getting the error " 'BatchNorm2d' object has no attribute 'track_running_stats'' for both the binary and multiclassifier code. I think this has to do with the version of PyTorch.
